theme/stylesheets/website/summary.less

```diff
@@ -32,6 +32,7 @@
 
             li {
                 list-style: none;
+                font-size: @s-font-size;
```

Comment:
I think, based on the variables I see here: https://github.com/mautic/documentation/blob/master/theme/stylesheets/website/variables.less
that this should be @font-size-small?

Reply:
```less
@s-font-size:                         1.2rem;
@l-font-size:                         1.6rem;
@default-font-size:                   @l-font-size;
```

At the bottom of the same file is where my tracing got me, as the inheritance goes all the way to `@default-font-size`.

However, `@font-size-small` all nice and prominent at the top of the file, does seem more user-friendly than `@s-font-size`